postgre-client/src/load_client.rs

```diff
@@ -131,11 +140,10 @@ impl PgClient {
         transaction: &mut Transaction<'_, Postgres>,
     ) -> Result<(), IndexDbError> {
         for (table, constraint) in [("assets_v3", "assets_v3_authority_fk_constraint")] {
-            let mut query_builder: QueryBuilder<'_, Postgres> = QueryBuilder::new("ALTER TABLE ");
-            query_builder.push(table);
-            query_builder.push(" DROP CONSTRAINT ");
-            query_builder.push(constraint);
-            query_builder.push(";");
+            let mut query_builder: QueryBuilder<'_, Postgres> = QueryBuilder::new(format!(
+                "ALTER TABLE {} DROP CONSTRAINT {};",
+                table, constraint
```

Comment:
the primary reason why push was used instead of format was the possibility of injection with format. We're more or less protected by the client not allowing multiple statements in a single query though